packages/common-i18n/src/locales/fi/search.json

```diff
@@ -60,9 +60,20 @@
       "minAge": "{{age}} {{yearAbbr}}",
       "maxAge": "{{age}} {{yearAbbr}}",
       "exactAge": "{{age}}-vuotiaalle"
-    }
+    },
+    "labelAccessibilityProfile": "Esteettömyys",
+    "ariaLabelClearAccessibilityProfile": "Poista esteettömyys valinta"
```

Comment:
`esteettömyys valinta` is a compound word i.e. `esteettömyysvalinta` (Also thought that could it be `esteettömyys-valinta`, but kielitoimisto's guides [#1](http://www.kielitoimistonohjepankki.fi/ohje/80) & [#2](http://www.kielitoimistonohjepankki.fi/ohje/130) would suggest that would not be so).